README.md

```diff
@@ -1,14 +1,22 @@
-# Website for Generating a MATLAB Workflow
+# Generate Starter Workflow for MATLAB
 
-The [GitHub Actions Workflow Generator for MATLAB](https://matlab-actions.github.io/workflow-generator/) website enables you to generate a starter GitHub&reg; Actions workflow for your GitHub repository that contains MATLAB&reg; code and Simulink&reg; models.
+Use [GitHub&reg; Actions Workflow Generator for MATLAB](https://matlab-actions.github.io/workflow-generator/) to create a starter workflow for a repository that contains MATLAB&reg; code and Simulink&reg; models. You can use the generated workflow as a foundation for defining your own custom workflow.
 
-## See Also
+To generate a starter workflow for your repository, in the workflow generation interface:
+
+1. Enter either your GitHub username followed by your repository name, or provide the full URL of your repository. 
```

Comment:
Can we show the username/repository format here. I feel like folks could interpret this to mean username SPACE repository name.

Reply:
Ok, I like this better. It should be easier to copy the URL, so it can appear first.